glslc/src/file_compiler.h

```diff
@@ -69,12 +72,45 @@ class FileCompiler : public shaderc_util::Compiler {
   void SetIndividualCompilationMode();
 
   // Instead of outputting object files, output the disassembled textual output.
-  virtual void SetDisassemblyMode() override;
+  virtual void SetDisassemblyMode();
+
+  // Delegates CompilerOptions to add macro definiations.
```

Comment:
Typo "definitions".  Also not clear what is being delegated to whom.  Consider rewriting in simpler terms.